### PR TITLE
Update Telligent Community Provider

### LINKED ISSUE #122: Update Community Server Provider
"Community Server" appears as an option in the Blog Providers list.

"Community Server" has been re-branded under different names since version 5.x (released around 2009/10), primarily as "Telligent Community" (or some form of this), for a brief period it was known as "Zimbra Social (version 8.x)", and is now back to Telligent.

In version 8.x the URLs for the metablog API were updated, this, along with the old name, make the process of setting up a Telligent blog confusing.

Thinking that "Community Server" should be updated to the more widely used "Telligent Community", and an extra provider added with the new URL structure for version 8.x+. How does that sound?

I've given the changes a go in this branch: https://github.com/rhysgodfrey/OpenLiveWriter/tree/update-telligent-provider, but would be good to get some guidance on if the changes are wanted and acceptable before opening a PR :)

### PR DESCRIPTION
**Pull request by rhysgodfrey**
Telligent Community hasn't been called "Community Server" since version
4.x. The metablog URL was updated in 8.x so the suggested URL when
picking "Community Server" no longer works.

- Updating the naming of "Community Server" to "Telligent Community"
- Adding new provider for "Telligent Community 8.0+" with updated URL
pattern and supporting in all markets.

Fixes #122

### CODE REVIEW

**Comment by ScottIsAFool** [comment]
LGTM

File: src/managed/OpenLiveWriter.BlogClient/Providers/BlogProvidersB5.xml [hunk 1]
```diff
@@ -340,6 +340,36 @@
...
+		http://<hostname>/<groupname>/b/<blogname>/metablog
+		]]>
+    </postApiUrl>
+    <rsdEngineNamePattern>Community Server</rsdEngineNamePattern>
+    <options>
+      <supportsNewCategories>Yes</supportsNewCategories>
+      <supportsNewCategoriesInline>Yes</supportsNewCategoriesInline>
+      <useLocalTime>Yes</useLocalTime>
+      <requiresHtmlTitles>No</requiresHtmlTitles>
+      <returnsHtmlTitlesOnGet>Yes</returnsHtmlTitlesOnGet>
```
Comment (willduff) [inline src/managed/OpenLiveWriter.BlogClient/Providers/BlogProvidersB5.xml]: Are you able to test some of these options against a Telligent Community 8.0+ install to see if they're still applicable to the latest version?

* Make sure publishing a post with images works (`fileUploadNameFormat`)
* Try creating at least two new categories via Open Live Writer and then publish the post. (`supportsNewCategories`, `supportsNewCategoriesInline`, `supportsMultipleCategories`)
* Try scheduling a post for a future date and time (even if its just 10min in the future) and make sure the timing works (`useLocalTime`)
* Publish a blog post where the title has angle brackets `<` in the title, then edit the post in Writer and then publish it again and make sure that the post looks correct on both the blog and in Writer (`requiresHtmlTitles` and `returnsHtmlTitlesOnGet`)
* Start a new blog post and publish as draft (`supportsPostAsDraft`)
* If you have time, you could flip <supportsEmbeds> and <supportsScripts> to Yes and try inserting a YouTube video (`supportsEmbeds`) and try inserting a GitHub gist (`supportsScripts`) and publishing that. It's possible that the policies have changed.

Thanks! If not, we can try and help recruit someone to test this.

**Comment by rhysgodfrey** [comment]
Hi @willduff,

Run the suggested tests against an instance of Telligent 8.5 and they are all fine. Although did encounter #96 when running using the "Debug" build configuration and publishing the post, but this didn't affect the posting, running using "Release" build configuration is fine.

Tried the `supportsEmbeds` and `supportsScripts` flags and it still does not work, as standard Telligent strips out all script tags. Embedding You Tube videos is possible but you need to use a specific syntax.

Thanks,
Rhys

**Comment by RobDolinMS** [comment]
THANK YOU @rhysgodfrey for this contribution.  Sorry review has taken so long.

**Pull request merged**